src/jbms/argparse.cpp

```diff
@@ -5,11 +5,12 @@
 #include <cctype>
 #include <iostream>
 
-// Needed to determine terminal width
-// TODO: add support for Windows
-#include <sys/ioctl.h>
-#include <stdio.h>
-#include <unistd.h>
+#if defined(_WIN64) || (defined(__CYGWIN__) && !defined(_WIN32))
+  #include <windows.h>
+#elif defined(__linux__)
```

Comment:
I think this should just be `#else` since this will also work on BSD, macOS.

Reply:
I'm sorry I can add support for macOS as well but don't know anything about BSD systems yet. If `ioctl` works on BSD's system then `#else` would be a better option @jbms what do you say?

Reply:
Yes `ioctl` should work on BSD.